content/en/docs/installation/system-requirements.md

```diff
@@ -0,0 +1,85 @@
+
+Armory Enterprise can be installed on any [certified Kubernetes cluster](https://www.cncf.io/certification/software-conformance/) that meets the following version requirements:
+
+* **Minimum version**: 1.16
+* **Maximum version**: 1.20
+<!-- track EKS versions -->
+
+You install Armory Enterprise using the Armory Operator (a Kubernetes operator), which has the following requirements:
+
+- You must be able to apply Kubernetes manifests and CRDs, either directly using `kubectl` commands from your machine or another method.
+- By default, the Operator pulls images from a public registry. If you cannot pull images from public registries, see {{< linkWithTitle "ag-operator.md" >}}.
+
+Note that Armory does not produce marketplace specific images that can be used by different certified Kubernetes offerings. 
+
+## Browsers
+
+The UI for Armory Enterprise works best on Firefox or Chromium-based browsers.
+
+## External storage
+
+Armory Enterprise requires external storage for storing metadata and history.
+
+### Bucket storage
+
+You need an S3-compatible object store, such as an S3 bucket or Minio, for persisting your application settings and pipelines. The account you use to install and run Armory Enterprise needs read/write access to the buckets.
+
+### RDBMS (SQL)
+
+Depending on the service, Armory Enterprise also uses either Redis, MySQL, or Postgres as a backing store. The following table lists the supported database and the  service:
+
+| Database | DB version             | Armory                 | Spinnaker services                                  | Note                                                                                                                       |
+| -------- | ---------------------- | ---------------------- | --------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| Redis    | All supported versions | All supported versions | All Spinnaker services that require a backing store | The DB versions refer to external Redis instances. Supported only for services that still require Redis for a feature, such as Gate sessions. Redis is not supported as a core persistent storage engine. Although Spinnaker deploys internal Redis instances, do not use these instances for production deployments. Armory recommends only using them for testing and proof-of-concept deployments. |
+| MySQL    | MySQL 5.7 (or Aurora)  | All supported versions | Clouddriver, Front50, Orca                          |                                                                                                                            |
+| PostgreSQL    | PostgreSQL 10.0 or later  | 2.24.0 or later | Clouddriver                          |                                                                                                                            |
```

Comment:
MySQL and PostgreSQL version are now parameters in config.toml.
mysql-version="5.7; AWS Aurora"
postgresql-version="10+"

Reply:
Cool! Thanks for thinking ahead